google/cloud/forseti/services/cli.py

```diff
@@ -1193,5 +1196,4 @@ def get_config_path():
 
 
 if __name__ == '__main__':
-    ENV_CONFIG = DefaultConfigParser.load()
-    main(sys.argv[1:], ENV_CONFIG)
```

Comment:
qq: why is this change necessary?

Reply:
I just want to keep it consistent as the server and the enforcer both have their main() methods self contained.